ingestion/functions/parsing/mexico/mexico.py

```diff
@@ -160,6 +162,16 @@ def parse_cases(raw_data_file: str, source_id: str, source_url: str):
     with open(raw_data_file, "r") as f:
         reader = csv.DictReader(f)
         for row in reader:
+            classification = parsing_lib.safe_int(row.get('CLASIFICACION_FINAL', None))
+            # 1	CASO DE COVID-19 CONFIRMADO POR ASOCIACIÓN CLÍNICA EPIDEMIOLÓGICA
+            # 2	CASO DE COVID-19 CONFIRMADO POR COMITÉ DE  DICTAMINACIÓN
+            # 3	CASO DE SARS-COV-2  CONFIRMADO
+            # 4	INVÁLIDO POR LABORATORIO
+            # 5	NO REALIZADO POR LABORATORIO
+            # 6	CASO SOSPECHOSO
+            # 7	NEGATIVO A SARS-COV-2
+            if classification is None or classification not in [1, 2, 3]:
```

Comment:
`classification not in [1, 2, 3]` includes `classification is None`, so the first test is redundant.